packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -78,6 +80,18 @@ class FlutterPlugin implements Plugin<Project> {
                     matchingFallbacks = ['debug', 'release']
                 }
             }
+            dynamicProfile {
+                initWith debug
+                if (it.hasProperty('matchingFallbacks')) {
+                    matchingFallbacks = ['debug', 'release']
+                }
+            }
+            dynamicRelease {
```

Comment:
comment above at line 75 seems to need to be updated

Reply:
Done.